src/report/superset-report.controller.spec.js

```diff
@@ -15,36 +15,61 @@
 
 describe('SupersetReportController', function() {
 
+    var vm, $controller, $q, $rootScope, reportCode, reportUrl,
+        supersetLocaleService, SUPERSET_URL;
+
     beforeEach(function() {
-        module('report');
+        SUPERSET_URL = 'http://localhost/superset';
+        reportCode = 'reportCode';
+        reportUrl = SUPERSET_URL + '/theReport';
```

Comment:
You should assign all of those variables to 'this' to avoid memory leaks.

Reply:
Done.